woocommerce/woocommerce-gutenberg-products-block
# Store Customization > Fix the Testimonials 3 Columns pattern

## What

The changes on this PR include:

* Fix for a PHP warning that could be triggered if certain parts of the content for the pattern don't exist on the AI-generated content within the `'wc_blocks_patterns_content'` option yet.
* Fix alignment for the title and remove padding for this pattern

## Why



This change is necessary to ensure no warnings will be thrown and the design matches the expected 

Figma: Mrk6SERPZ4KrFHSjM0a8TK-fi-5716_46688

## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Create a new post
2. Insert the Testimonials 3 Columns pattern
3. Make sure it works as expected both on the editor side and on the front end
4. Check your PHP error log and confirm you don't see any errors related to this pattern in particular.


## Screenshots or screencast



| Before | After |
| ------ | ----- |
|  <img width="1278" alt="Screenshot 2023-10-06 at 08 45 52" src="https://github.com/woocommerce/woocommerce-blocks/assets/15730971/ce3eadb2-9109-4649-b4df-0980bbdb4497"> |  <img width="1253" alt="Screenshot 2023-10-06 at 08 46 02" src="https://github.com/woocommerce/woocommerce-blocks/assets/15730971/53db772f-653e-4fc8-b5bb-d6515b2dc8cf"> |

## WooCommerce Visibility


Required:


## Checklist

Required:

Conditional:

## Changelog


> Testimonials 3 Columns pattern > Update the width and fix the PHP warnings that could be triggered if the content saved within the wc_blocks_patterns_content option didn't match the updated patterns dictionary.
